core/domain/rights_manager.py

```diff
@@ -379,7 +379,8 @@ def _can_edit(self, rights_object):
         return (
             self._has_editing_rights(rights_object) or (
                 self.is_moderator() and (
-                    rights_object.status != ACTIVITY_STATUS_PRIVATE)
+                    rights_object.status != ACTIVITY_STATUS_PRIVATE) or
+                self.is_admin()
```

Comment:
Could you move this before the complex clause? Hard to remember which of and/or takes precedence.

I.e.

    self._has_editing_rights(rights_object) or self.is_admin() or (...)